operate/utils/__init__.py

```diff
@@ -45,3 +46,50 @@ def create_backup(path: Path) -> Path:
         shutil.copy2(path, backup_path)
 
     return backup_path
+
+
+NestedDict = t.Union[int, t.Dict[str, "NestedDict"]]
+
+
+def merge_sum_dicts(*dicts: NestedDict) -> NestedDict:
```

Comment:
This is not used in this PR, but will be used to simplify logic in several parts of the code in upcoming PRs.